source/fundamentals/logging.txt

```diff
@@ -0,0 +1,285 @@
+=======
+Logging
+=======
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _logging:
+
+Overview
+--------
+
+.. What is on this page?
+
+In this guide, we show you how to set up and configure a logger in the
+MongoDB Java driver.
+
+.. What do any new terms mean?
+
+.. Not applicable. We are treating definition of logger as understood.
+
+.. What can you expect to see on this page?
+
+You will learn how to:
+
+- Set up a logger using the Simple Logging Facade For Java (SLF4J)
+- Configure the log level of your logger
+
+.. Who should read this? Where should I go if this isn't the page I was looking for?
+
+.. Not Applicable. No natural redirect location.
+
+Set Up a Logger
+---------------
+
+In this section, we provide background on the dependencies necessary to set up a
+logger and walk through an example logger setup. 
```

Comment:
Is this sentence needed here or are the headings self-explanatory?

Reply:
I rewrote this section, so response is relative to rewrite: https://github.com/mongodb/docs-java/pull/76/commits/d7007eaa9fe51be1362c4c207773a6d823be4f27

Thanks for the note. After rereading this a few times, I think the sentence serves as a useful transition, but my instincts could be incorrect. I think "background on the dependencies" is meaningfully different than "background", and "example of a logger setup" is different than "example". That being said, if you disagree, I would appreciate a discussion of what you feel would improve the section as I am having trouble seeing the issue. 

I am trying to balance the following things: 

1. Smooth Transition into Section
2. Heavy use of heading structure
3. Avoid [stacking headings](https://docs.mongodb.com/meta/style-guide/style/titles-and-headings/)